app-ios/Native/Sources/Component/TimetableComponents.swift

```diff
@@ -27,14 +27,14 @@ public struct RoomTag: View {
     }
 
     private var textColor: Color {
-        if room.type == .roomH {
+        if room.type == .roomL {
             return AssetColors.surface.swiftUIColor
         }
         return room.roomTheme.primaryColor
     }
 
     private var backgroundColor: Color {
-        if room.type == .roomH {
+        if room.type == .roomL {
```

Comment:
According to Figma, what should be specified here is roomJ and roomK, but in terms of resource colors, it seems only roomK can fit this design. Fine-tuning might be out of scope for this PR, but it does look like some adjustment to RoomTag would be beneficial.

<img width="478" height="407" alt="Image" src="https://github.com/user-attachments/assets/38565feb-e74b-4ce7-8e3f-a9a9995b630f" />

<img width="179" height="133" alt="Image" src="https://github.com/user-attachments/assets/f9edb885-6791-4c9a-81b6-f92dfab4970b" />

Reply:
<img width="299" height="631" alt="image" src="https://github.com/user-attachments/assets/17a8966f-24a1-4b8e-bdd0-8e7d4d39d14b" />